eldritch/test_events.py

```diff
@@ -107,6 +107,34 @@ def testMoveOneSpace(self):
     self.assertEqual(self.char.place.name, "Easttown")
     self.assertEqual(self.char.movement_points, 3)
 
+  def testMoveOneSpaceToMonster(self):
+    cultist = next(monster for monster in self.state.monsters if monster.name == "Maniac")
+    cultist.place = self.state.places["Easttown"]
+    movement = Movement(self.char, [self.state.places["Easttown"]])
+    self.assertFalse(movement.is_resolved())
+    self.assertEqual(self.char.movement_points, 4)
+
+    self.state.event_stack.append(Sequence([movement, EndMovement(self.char)], self.char))
+    choice = self.resolve_to_choice(MultipleChoice)
+    self.assertEqual(choice.choices, ["Fight", "Evade"])
+    choice.resolve(self.state, "Fight")
+
+    next_choice = self.resolve_to_choice(MultipleChoice)
+    self.assertEqual(next_choice.choices, ["Fight", "Flee"])
+    next_choice.resolve(self.state, "Fight")
+
+    third_choice = self.resolve_to_choice(CombatChoice)
+    self.assertIsNone(third_choice.choices)
+    third_choice.resolve(self.state, [])
+    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=5)):
+      self.resolve_until_done()
+
+    self.assertTrue(movement.is_resolved())
+    self.assertEqual(self.char.place.name, "Easttown")
+    # self.assertEqual(self.char.movement_points, 0)
```

Comment:
Why are you not asserting on the movement points here? I know monster trophies may not be complete, but you set movement points to 0 correctly, right?